instrumentation/aws-sdk/aws-sdk-1.11/testing/src/main/java/io/opentelemetry/instrumentation/awssdk/v1_11/AbstractS3ClientTest.java

```diff
@@ -82,12 +83,12 @@ private static Stream<Arguments> provideArguments() {
             "CreateBucket",
             "PUT",
             (Function<AmazonS3, Object>) c -> c.createBucket("testbucket"),
-            ImmutableMap.of("aws.bucket.name", "testbucket")),
+            Collections.singletonList(createStringKeyPair("aws.bucket.name", "testbucket"))),
```

Comment:
Recommend to use static import here, for detail, you can refer to https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/style-guideline.md#static-imports

Reply:
Sure